lib/dolly/request.rb

```diff
@@ -88,6 +92,10 @@ def values_to_json hash
     def full_path resource
       "/#{database_name}/#{resource}"
     end
+
+    def attachment_path resource, attachment_name
+      "#{full_path(resource)}/#{attachment_name}"
```

Comment:
Also to me it makes sense to encode here, since it is the method responsible for generating the path.

Lastly, feel free to ignore me since I have no idea what's going on anyway. :trollface: